Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -3471,11 +3471,12 @@ struct WebCore::CookieRequestHeaderFieldProxy {
     WebCore::IncludeSecureCookies includeSecureCookies;
 };
 
-[RefCounted, CustomMemberLayout] class WebCore::FormData {
-    bool m_alwaysStream;
-    Vector<char> m_boundary;
+[RefCounted] class WebCore::FormData {
     Vector<WebCore::FormDataElement> m_elements;
     int64_t m_identifier;
```

Comment:
It seems identifier should be a uint64_t if we change FormData::create to take a uint64_t.